### PR TITLE
[sophora-server]: add container ports to values.yaml (#108)

### PR DESCRIPTION
**Pull request by hehu80**

### CODE REVIEW

File: charts/sophora-server/templates/statefulset.yaml [hunk 1]
```diff
@@ -272,21 +272,21 @@ spec:
...
-            - containerPort: 1199
+            - containerPort: {{ .Values.sophora.server.ports.rmiRegistry }}
               name: rmi-registry
           {{- end }}
           {{- if .Values.livenessProbe.enabled }}
           livenessProbe:
             httpGet:
               path: /status/liveness
-              port: 1196
+              port: {{ .Values.sophora.server.ports.http }}
```
Comment (tom-schoener) [inline charts/sophora-server/templates/statefulset.yaml:289]: I'd rewrite the port as `http` for all probes:

```
port: http
```

**Review by tom-schoener — CHANGES_REQUESTED** [review]
One minor refactoring of the port reference. When fixed, I'll approve and merge the request.

**Comment by hehu80** [comment]
@tom-schoener can you take another look, we urgently need the change

**Comment by tom-schoener** [comment]
That is not what I had in mind. Sorry for the confusion. I meant the `port` of the probes in `httpGet`: 

```yaml
livenessProbe:
  httpGet:
    path: /status/liveness
    port: http # <-----
```

The variable name `ports` was fine.

File: charts/sophora-server/templates/statefulset.yaml [hunk 2]
```diff
@@ -272,21 +272,21 @@ spec:
...
-            - containerPort: 1199
+            - containerPort: {{ .Values.sophora.server.port.rmiRegistry }}
               name: rmi-registry
           {{- end }}
           {{- if .Values.livenessProbe.enabled }}
           livenessProbe:
             httpGet:
               path: /status/liveness
-              port: 1196
+              port: {{ .Values.sophora.server.port.http }}
```
Comment (tom-schoener) [inline charts/sophora-server/templates/statefulset.yaml:289]:
```suggestion
              port: http
```

File: charts/sophora-server/templates/statefulset.yaml [hunk 3]
```diff
@@ -296,7 +296,7 @@ spec:
           readinessProbe:
             httpGet:
               path: /status/ready
-              port: 1196
+              port: {{ .Values.sophora.server.port.http }}
```
Comment (tom-schoener) [inline charts/sophora-server/templates/statefulset.yaml:299]:
```suggestion
              port: http
```

File: charts/sophora-server/templates/statefulset.yaml [hunk 4]
```diff
@@ -306,7 +306,7 @@ spec:
           startupProbe:
             httpGet:
               path: /status/started
-              port: 1196
+              port: {{ .Values.sophora.server.port.http }}
```
Comment (tom-schoener) [inline charts/sophora-server/templates/statefulset.yaml:309]:
```suggestion
              port: http
```

**Review by tom-schoener — CHANGES_REQUESTED** [review]

**Comment by hehu80** [comment]
@tom-schoener can you, again, take another look, we urgently need the change

**Review by tom-schoener — APPROVED** [review]

**Pull request merged**